lib/Wikia/src/Persistence/User/Preferences/PreferencePersistence.php

```diff
@@ -228,9 +232,24 @@ private function findWikisWithLocalPreference( ReverseLookupApi $api, $preferenc
 		return $wikiList;
 	}
 
-	private function findUsersWithGlobalPreference( ReverseLookupApi $api, $preferenceName, $value = null ) {
-		$userList = $api->findUsersWithGlobalPreference( $preferenceName, $value );
-		return $userList;
+	/**
+	 * Calls user-preference /reverse-lookup/global/{preferenceName}/users endpoint to get an array of string userIds
+	 * of users with $preferenceName set to $value (or set at all if $value is null)
+	 * @param ReverseLookupApi $api
+	 * @param string $preferenceName
+	 * @param string|null $value
+	 * @param int $limit - must be less than set REVERSE_LOOKUP_GLOBAL_USERS_MAX_LIMIT
+	 * @param int|null $user_id_continue
+	 * @return array|string[] - userId
+	 * @throws ApiException
```

Comment:
is it allowed in PHP to specify in such comments the class of the exception and the condition when the methods throws it?
Like in Java:
```java
/**
@throws IllegalArgumentException when the requested number of users exceeds the user limit {@value #REVERSE_LOOKUP_GLOBAL_USERS_MAX_LIMIT}
*/
```